static/js/modules/helpers.js

```diff
@@ -180,6 +213,9 @@ module.exports = {
   filterNull: filterNull,
   buildAppUrl: buildAppUrl,
   buildUrl: buildUrl,
+  isMediumScreen: isMediumScreen,
+  isLargeScreen: isLargeScreen,
   globals: globals,
-  getTimePeriod: getTimePeriod
+  getTimePeriod: getTimePeriod,
+  zeroPad: zeroPad
```

Comment:
Not a big deal, maybe alphabetize this list so it's easier to spot duplicates and find what you're looking for?